apps/projects/migrations/0006_initialize_insights.py

```diff
@@ -59,7 +58,7 @@ def initialize_insights(apps, schema_editor):
 
         if model_name == "comment":
             project = content_object.project
-        elif model_name in ["idea", "map_idea", "topic", "proposal"]:
+        elif model_name in ["idea", "mapidea", "topic", "proposal"]:
             project = content_object.module.project
         else:
             print(f"could not identify project: {rating=}")
```

Comment:
for consistency: as we started using the logger now: maybe use logger.log / or logger.warn here?

Reply:
github won't let me comment on line 5: the import of Q is unused and can be removed

Reply:
done